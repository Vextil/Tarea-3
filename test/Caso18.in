# stress Solicitudes
# Prueba de orden de tiempo
TiempoSolicitudes
# Prueba de uso de memoria
MemoriaSolicitudes
Fin
