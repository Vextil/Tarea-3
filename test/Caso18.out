1># stress Solicitudes
2># Prueba de orden de tiempo
3>Fin de la prueba. 
4># Prueba de uso de memoria
5>Fin Destruir.Fin Listar.Fin Operaciones.
6>Fin
